lib/tests-api/actions-builder.js

```diff
@@ -392,7 +392,7 @@ function replaceStack(error, stack) {
 }
 
 function serializeFunc(func) {
-    return '(' + func.toString() + '(window));';
+    return '(' + func.toString() + ').apply(null, [window].concat(Array.prototype.slice.call(arguments)));';
```

Comment:
No particular reason. `null` should get replaced with the global object, which in the browser is `window`, so I believe they're equivalent. I can change it to `window`.